libimagtodo/src/delete.rs

```diff
@@ -0,0 +1,30 @@
+// Needed for reading a Json File
+// extern crate rustc_serialize;
+// use rustc_serialize::json::Json;
+// use std::fs::File;
+// use std::io::Read;
+
+use std::ops::Deref;
+use toml::Value;
+use uuid::Uuid;
+
+use libimagstore::store::Store;
+use libimagstore::storeid::IntoStoreId;
+use module_path::ModuleEntryPath;
+
+/// With the uuid we get the storeid and than we can delete the entry
+fn deleteFunc(uuid: Uuid, store : &Store) {	
+	// With this we can read from a .json File
+	// let mut file = File::open("text.json").unwrap();
+	// let mut data = String::new();
+	// file.rad_to_string(&mut data).unwrap();
+	//
+	// let jeson = Json::from_str(&data).unwrap();
+	// println!("{}", json.find_path(&["uuid"]).unwrap());		
+
+	// With the uuid we get the storeid
+	let store_id = ModuleEntryPath::new(format!("taskwarrior/{}", uuid)).into_storeid();
+	// It deletes an entry	
+	store.delete(store_id);
```

Comment:
store.delete(...) returns a Result. Wrap the Error in a new TodoError (with a new TodoErrorKind) and return that on Error, or None as Result